tools/SourceKit/CMakeLists.txt

```diff
@@ -234,6 +238,11 @@ macro(add_sourcekit_executable name)
       set_target_properties(${name}
         PROPERTIES
         LINK_FLAGS "-Wl,-exported_symbol,_main")
+
+      if(SWIFT_ANALYZE_CODE_COVERAGE)
+        set_property(TARGET "${name}" APPEND_STRING PROPERTY
```

Comment:
Right, but what I'm saying is that `_add_variant_c_compile_link_flags` should return these two flags, and they will be added to `LINK_FLAGS` by other machinery in `AddSwift.cmake`.